libs/javascript/@local/eslint-config/generate-block-config.cjs

```diff
@@ -0,0 +1,16 @@
+/**
+ * @param {string} workspaceDirPath
+ * @returns {import("eslint").Linter.Config}
+ */
+module.exports = (workspaceDirPath) => ({
+  root: true,
+  extends: [
+    "@local/eslint-config/legacy-base-eslintrc-to-refactor.cjs",
+    "@local/eslint-config/legacy-block-eslintrc-to-refactor.cjs",
```

Comment:
These two files mimic the contents of formerly used base configs. Keeping them as is for now till we find time to look into rules themselves.